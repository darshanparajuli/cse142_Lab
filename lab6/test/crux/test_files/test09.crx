// test addition

func main() : void {
  ::printInt(1+2);
  ::println();
  ::printFloat( 0.5-0.125);
}
