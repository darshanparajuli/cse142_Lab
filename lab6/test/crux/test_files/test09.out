3
0.37500000